packages/elements/src/list/helpers/id.ts

```diff
@@ -1,10 +1,10 @@
 /**
- * Combines prefix and a string to give unique item key
+ * Combines prefix and a string to give unique item id used by aria-activedescendant
  * @param prefix a unique randomly generated string
  * @param value a string of valid item value
- * @returns unique item key
+ * @returns a unique item id
  */
-const getItemKey = (prefix: string, value: string): string => {
+const getItemId = (prefix: string, value: string | undefined): string => {
   const SEPARATOR = '-';
```

Comment:
Can we remove this const and just place the separator in the string? You probably want to rename the file, too.`item-id.ts`?

Reply:
Thanks, agreed and fixed.